task/types.go

```diff
@@ -14,14 +12,14 @@ type Task struct {
 	Type string `json:"type"`
 
 	// Data provides task execution data.
-	Data json.RawMessage `json:"data"`
+	Data []byte `json:"data"`
 
 	// Driver provides the execution driver used to
 	// execute the task.
 	Driver string `json:"driver"`
 
 	// Config provides the execution driver configuration.
-	Config json.RawMessage `json:"config"`
+	Config []byte `json:"config"`
```

Comment:
after this PR merges, pl. update the go.mod in the runner repo.
Currently runner doesn't point to the main branch of the go-task

Reply:
Will do!